src/template.nunjucks

```diff
@@ -0,0 +1,185 @@
+FORMAT: 1A
+{% if api.meta.HOST %}HOST: {{ api.meta.HOST }}{% endif %}
+
+
+
+
+
+{% macro renderExample(example) %}
+{% for copy in example|getCopy %}
+{{ copy.content }}
+{% endfor %}
+
+{% if example|bodyOnly %}
+        {{ example.messageBody.content|indent(8) }}
+{% else %}
+  {% if example.headers and example.headers.exclude('Content-Type').length %}
+    + Headers
+
+      {% for header in example.headers.exclude('Content-Type').content %}
+            {{ header.key.toValue() }}: {{ header.value.toValue() }}{% endfor %}
+
+  {% endif %}
+  {% if example.dataStructure %}
+    {{ example.dataStructure.content|mson|indent(4) }}
+  {% endif %}
+  {% if example.messageBody %}
+    + Body
+
+            {{ example.messageBody.content|pretty|indent(12) }}
+
+  {% endif %}
+  {% if example.messageBodySchema %}
+    + Schema
+
+            {{ example.messageBodySchema.content|pretty|indent(12) }}
+
+  {% endif %}
+{% endif %}
+{% endmacro %}
+
+
+
+
+
+
+{% macro renderParameters(hrefVariables) %}
++ Parameters
+
+  {% for item in hrefVariables.content %}
+    {% set typeAttributes = item.attributes.get('typeAttributes') %}
+    + {{ item.key.toValue() }}{% if typeAttributes %} ({{ typeAttributes.toValue() }}){% endif %}{% if item.value.description %} - {{ item.value.description }}{% endif %}{% endfor %}
```

Comment:
Latest push adds support for parameter sample values. Hopefully in the future this code path can be merged with the full MSON code.